conda_forge_tick/update_upstream_versions.py

```diff
@@ -24,14 +24,20 @@
 logger = logging.getLogger("conda-forge-tick._update_versions")
 
 
-def verify(version: str, next_version: str):
+def version_match(version, new_version: str, payload):
     """This function takes two versions strings and compare them.
-    If either of these match, then the version bump is performed if they don't match, then it won't."""
-
-    pattern = r"\d+\.\d+\.[02468]"
-    split_version = re.match(pattern, next_version)
-    if split_version.group() > version:
-        return True
+        If either of these match, then the version bump is performed if they don't match, then it won't."""
+    # Example pattern \d+\.\d+\.[02468] TODO need to add an standard version parser when there isn't any given pattern
+
+    yml = payload.__getitem__('conda-forge.yml')
+    with yml['bot'] as bot:
```

Comment:
I doubt that this works, since the dict inside the payload is just a dict not lazyjson

Reply:
Yeah I've checked it now, sorry for the mistake.

Reply:
I'm also going to insert a 'N version' checker too, what do you think ? (i.e Bump the version every N versions)